### ISSUE TITLE
Add scikit-learn to install dependencies

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#147]
Maybe we should add sklearn to install dependecies as well? It's required for LIME that can be used to explain non-sklearn based estimators, and currently top-level eli5 function will also fail to import without scikit-learn installed.

**Comment by kmike** [comment]
scikit-learn is in install docs - http://eli5.readthedocs.io/en/latest/overview.html#installation. I haven't put scikit-learn to `install_requires` because (if I recall correctly) installing both numpy/scipy and scikit-learn from the same install_requries doesn't work. But maybe this is no longer the case, and we can add scikit-learn to install_requires :)

**Comment by lopuhin** [comment]
Ah, right, I remember some libraries had problem with it, let me check.

**Comment by kmike** [comment]
We can also do the opposite - add scikit-learn, but remove numpy/scipy

**Comment by lopuhin** [comment]
Just tested installation with pip 7.1.2 with and without wheels - it works if scikit-learn and numpy/scipy are in install requirements (I did pip install of a local tar.gz package built with python sdist). I'd rather leave them all there, because we are using numpy and scipy directly as well.

**Comment by kmike** [comment]
I'm fine with that 👍
